kolibri/plugins/epub_viewer/assets/src/views/CustomThemeItem.vue

```diff
@@ -0,0 +1,135 @@
+<template>
+
+  <div style="margin: 16px 0 0 0;">
```

Comment:
Does this work in RTL languages? i.e. Arabic? We do have some handling for flipping lefts and rights with styling, but it might require using a class here, not just adding the style like this.

Reply:
Its only the top margin of the entire div, which is a row displayed in the settings sidebar. I will put it into a class anyway.